src/sections/home/Home.Hero.tsx

```diff
@@ -25,6 +25,10 @@ const HomeHero: React.FC = () => {
     return undefined
   }, [scrolled])
 
+  const scrollDown = () => {
+    window.scrollBy(0, window.innerHeight)
```

Comment:
would it be possible to make this scroll smooth (like how when you click on 'contact' it smoothly scrolls down

if that's not possible dw about it, i'll just approve the pr anyways

approved, ty for making short work of this!!